src/datasources/cache/redis.cache.service.ts

```diff
@@ -31,7 +31,13 @@ export class RedisCacheService
     return this.client.ping();
   }
 
-  async set(
+  async getCounter(key: string): Promise<number | null> {
+    const keyWithPrefix = this._prefixKey(key);
+    const value = await this.client.get(keyWithPrefix);
+    return value ? Number(value) : null;
```

Comment:
If the key returns a non-numeric string, this could be `NaN`. We should add a respective check. What do you think?

Reply:
Fixed in [1f478d0](https://github.com/safe-global/safe-client-gateway/pull/1948/commits/1f478d011fc9e89cdefdc5a961a7c2e1efb9bb66)

Thanks!